msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -720,3 +720,17 @@ You can define any of the following properties:
 * `developerUrl`: The HTTPS URL of the developer's website.
 * `privacyUrl`: The HTTPS URL of the developer's privacy policy.
 * `termsOfUseUrl`: The HTTPS URL of the developer's terms of use.
+
+## defaultBlockUntilAdminAction
+
+**Optional**—boolean
+ 
+When `defaultBlockUntilAdminAction` property is set to true, the app is hidden from users by default until Admin allows it. If set to **true**, the app is hidden for all tenants and end users. The tenant admins can see the app in the Teams admin center and take action to allow or block the app. The default value is **false**.
+
+## publisherDocsUrl
+
+**Optional** - string
+
+**Maximum size** - Characters
+
+The description that appears to the admins to indicate the purpose of this property.
```

Comment:
The URL to an information page where admin can get more guidelines before allowing a default blocked app.